src/azure-cli/azure/cli/command_modules/containerapp/_clients.py

```diff
@@ -16,7 +16,7 @@
 
 logger = get_logger(__name__)
 
-CURRENT_API_VERSION = "2023-05-01"
+CURRENT_API_VERSION = "2023-11-02-preview"
```

Comment:
Hi @lihaMSFT 

You cannot change the api-version to 2023-11-02-preview directly. This is a preview feature, please add in azure-cli-extensions like this PR: https://github.com/Azure/azure-cli-extensions/pull/7356